### ISSUE TITLE
project loading error

### ISSUE THREAD

**Issue by kyungsikjeung** [microchip-pic-avr-solutions/pic-iot-aws-sensor-node#8]
jump_table(.handle+0xb0): In function `.handle':
: undefined reference to `_spi1_close'
jump_table(.handle+0xb2): In function `.handle':
: undefined reference to `_spi1_close'
build/default/production/mcc_generated_files/drivers/spi_master.o(.const+0x0): undefined reference to `_spi1_close'
build/default/production/mcc_generated_files/drivers/timeout.o(.text.sortedInsert+0x20): In function `_sortedInsert':
: undefined reference to `_rebaseList'
build/default/production/mcc_generated_files/drivers/timeout.o(.text.sortedInsert+0x52): In function `.L9':
: undefined reference to `_setTimerDuration'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_open+0x3e): In function `_i2c_open':
: undefined reference to `_i2c2_clearIRQ'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_open+0x42): In function `_i2c_open':
: undefined reference to `_i2c2_driver_setBusCollisionISR'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_open+0x46): In function `_i2c_open':
: undefined reference to `_i2c2_driver_setMasterI2cISR'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_open+0x4a): In function `_i2c_open':
: undefined reference to `_i2c2_driver_setSlaveI2cISR'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_open+0x4c): In function `_i2c_open':
: undefined reference to `_i2c2_enableIRQ'
build/default/production/mcc_generated_files/drivers/timeout.o(.text.startTimerAtHead+0x40): In function `.L17':
: undefined reference to `_setTimerDuration'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_TX+0x0): In function `.LFB27':
: undefined reference to `_i2c2_driver_isNACK'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_TX+0x2a): In function `.L32':
: undefined reference to `_i2c2_driver_TXData'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_RX+0x4): In function `.LFB28':
: undefined reference to `_i2c2_driver_getRXData'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_RX+0x14): In function `.LFB28':
: undefined reference to `_i2c2_driver_sendACK'
build/default/production/mcc_generated_files/drivers/timeout.o(.text.timeout_create+0x16): In function `_timeout_create':
: undefined reference to `_makeAbsolute'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_masterOperation+0x1a): In function `.L66':
: undefined reference to `_i2c2_driver_start'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_masterOperation+0x1c): In function `.L66':
: undefined reference to `_i2c2_IRQisEnabled'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_masterOperation+0x26): In function `.L66':
: undefined reference to `_i2c_poller'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_TX_EMPTY+0x1c): In function `.L24':
: undefined reference to `_i2c2_setIRQ'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_ISR+0x0): In function `_i2c_ISR':
: undefined reference to `_i2c2_clearIRQ'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_ISR+0xa): In function `_i2c_ISR':
: undefined reference to `_i2c2_driver_isNACK'
build/default/production/mcc_generated_files/drivers/timeout.o(.text.timeout_isr+0x18): In function `_timeout_isr':
: undefined reference to `_enqueueCallback'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_close+0x14): In function `_i2c_close':
: undefined reference to `_i2c2_clearIRQ'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_close+0x16): In function `_i2c_close':
: undefined reference to `_i2c2_disableIRQ'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_SEND_ADR_WRITE+0xa): In function `.LFB21':
: undefined reference to `_i2c2_driver_TXData'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_SEND_ADR_READ+0xa): In function `.LFB20':
: undefined reference to `_i2c2_driver_TXData'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_RESET+0x0): In function `.LFB14':
: undefined reference to `_i2c2_driver_resetBus'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_RCEN+0x4): In function `.LFB22':
: undefined reference to `_i2c2_driver_startRX'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_DO_NACK_RESTART+0x0): In function `.LFB25':
: undefined reference to `_i2c2_driver_sendNACK'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_DO_NACK_STOP+0x0): In function `.LFB24':
: undefined reference to `_i2c2_driver_sendNACK'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_DO_ACK+0x0): In function `.LFB23':
: undefined reference to `_i2c2_driver_sendACK'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_SEND_STOP+0x0): In function `.LFB19':
: undefined reference to `_i2c2_driver_stop'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_SEND_RESTART+0x0): In function `.LFB18':
: undefined reference to `_i2c2_driver_restart'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_SEND_RESTART_WRITE+0x0): In function `.LFB17':
: undefined reference to `_i2c2_driver_restart'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.do_I2C_SEND_RESTART_READ+0x0): In function `.LFB16':
: undefined reference to `_i2c2_driver_restart'
build/default/production/mcc_generated_files/drivers/i2c_master.o(.text.i2c_busCollisionISR+0x0): In function `_i2c_busCollisionISR':
: undefined reference to `_i2c2_driver_clearBusCollision'
build/default/production/mcc_generated_files/application_manager.o(.text.runScheduler+0x0): In function `_runScheduler':
: undefined reference to `_timeout_callNextCallback'
make[2]: *** [dist/default/production/PICIoT.X.production.hex] Error 255
make[1]: *** [.build-conf] Error 2
make: *** [.build-impl] Error 2

**Comment by alexfanqi** [comment]
Probably their c compiler has some changes. For me, Just remove all inline from function signatures and it will work.

**Comment by zebestro** [comment]
Yeah, it looks like it, since I ran into the same thing recently. There seems to be some kind of issue with inline functions there.
